src/v2/Components/v2/ArtworkFilter/ArtworkFilters/MediumFilter.tsx

```diff
@@ -30,23 +31,25 @@ export const MediumFilter: FC = () => {
   return (
     <Toggle label="Medium" expanded={isExpanded}>
       <Flex flexDirection="column" alignItems="left">
-        {allowedMediums.map(({ value: slug, name }, index) => {
-          const selected =
-            currentFilters.additionalGeneIDs.includes(slug) ||
-            currentFilters.medium === slug
-          const props = {
-            key: index,
-            onSelect: selected => {
-              toggleMediumSelection(selected, slug)
-            },
-            selected,
-          }
-          return (
-            <Checkbox {...props}>
-              <OptionText>{name}</OptionText>
-            </Checkbox>
-          )
-        })}
+        <ShowMore>
+          {allowedMediums.map(({ value: slug, name }, index) => {
+            const selected =
+              currentFilters.additionalGeneIDs.includes(slug) ||
+              currentFilters.medium === slug
+            const props = {
+              key: index,
+              onSelect: selected => {
+                toggleMediumSelection(selected, slug)
+              },
+              selected,
+            }
```

Comment:
Just give me a single decent reason :)

Reply:
I think it's just personal pref. Happy to change it.

personally, my eyes are more drawn to 'code', in terms of variable assignments (`const checkbox, props, ...` etc.).

Then, the JSX fragment is always very 'clean' and I can just sort of glance through that.

vs. having a large/complex JSX fragment that sets lots of props, in addition to iteration and all that. I even like taking the `map` stuff out and moving those to a separate `renderGroup(..)` call to clean up the top level JSX even more (but i've stopped that in this PR after being called out earlier).

Reply:
It just defeats the purpose of JSX and is unconventional — like why not just go all the way and write direct `React.createElement` calls (don't do it).

It's not "clean" so much as it's indirect.

The render group stuff kills me too though — whenever I see something like that it's clear to me that the boundaries of the component could be improved and someone just doesn't want to make a new file or whatever.